indico/web/client/js/custom_elements/ind_date_picker.js

```diff
@@ -479,9 +479,18 @@ function toOptionalDate(dateString) {
 
 // Localization
 
+// This is used in browsers that do not support Intl API.
+// Only those items where the data differs from the generic
+// fallback are listed.
+const FALLBACK_WEEK_INFO = {
+  'en-US': {firstDay: 7, weekend: [6, 7]},
+  'pt-BR': {firstDay: 7, weekend: [6, 7]},
+};
+const GENERIC_FALLBACK_WEEK_INFO = {firstDay: 1, weekend: [6, 7]};
+
 function getWeekInfoForLocale(locale) {
-  let weekInfo = new Intl.Locale(locale).weekInfo;
-  weekInfo ??= {firstDay: 7, weekend: [6, 7]}; /* Firefox 😭 */
+  const weekInfo =
+    new Intl.Locale(locale).weekInfo ?? FALLBACK_WEEK_INFO[locale] ?? GENERIC_FALLBACK_WEEK_INFO;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/Locale/getWeekInfo

This sounds like both `.weekInfo` and `.getWeekInfo()` need to be tried?

Reply:
As far as I know, browsers that supported `.getWeekInfo()` all moved to `.weekInfo`, and those that don't support `.weekInfo` never supported `.getWeekInfo()` either.

Reply:
hm, I understood it the other way round that it moved from weekInfo to getWeekInfo... maybe worth double-checking

Reply:
https://caniuse.com/mdn-javascript_builtins_intl_locale_getweekinfo mentions `.weekInfo` being non-standard

Reply:
Interesting. When I was looking into this, I got the opposite idea. At any rate, I think generating this in Node (e.g., within the webpack config script) makes this problem go away, I think.

Reply:
Fine. But let's please NOT add more stuff to the webpack scripts, check the `preval` thing I mentioned in my other comment - it allows you to run some code at build-time w/o having to add more stuff to the config.

Reply:
Ok, will do.